src/.foedagenv_lin64.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+RETURN_PATH=`pwd`
+SCRIPT_PATH=`dirname $BASH_SOURCE`
+FOEDAG_PATH=`( cd "$SCRIPT_PATH" && pwd )`
+
+if [ -n "${LD_LIBRARY_PATH}" ]; then
+	export LD_LIBRARY_PATH=$FOEDAG_PATH/lib64/foedag/lib:$FOEDAG_PATH/lib64/tbb:$FOEDAG_PATH/lib/tbb:$FOEDAG_PATH/lib:$FOEDAG_PATH/lib/foedag/lib:$FOEDAG_PATH/bin/gtkwave/lib:$FOEDAG_PATH/external_libs/qt/lib:$FOEDAG_PATH/external_libs/lib:$LD_LIBRARY_PATH
```

Comment:
After $FOEDAG_PATH/lib, please also add $FOEDAG_PATH/lib64 as I have used cmake install variables so cmake creates a lib64 folder on CentOS and a lib folder in Ubuntu.

Reply:
Done